addons/sourcemod/scripting/PlayerManager.sp

```diff
@@ -566,7 +588,8 @@ public int Native_IsPlayerSteam(Handle hPlugin, int numParams)
 	}
 
 #if defined _Connect_Included
-	if (SteamClientAuthenticated(sAuthID32[client]))
+	bool bAuthenticated = GetFeatureStatus(FeatureType_Native, "SteamClientAuthenticated") == FeatureStatus_Available;
+	if (bAuthenticated && SteamClientAuthenticated(sAuthID32[client]))
```

Comment:
also if !bAuthenticated return true for steam because it will mean connect ext is disabled and server does not support nosteam